apis/python/src/tiledbsoma/common_nd_array.py

```diff
@@ -122,11 +122,13 @@ def build_tiledb_schema(
 
     cell_order, tile_order = create_options.cell_tile_orders()
 
+    # TODO: accept more TileDB array-schema options from create_options
+    # https://github.com/single-cell-data/TileDB-SOMA/issues/876
     return tiledb.ArraySchema(
         domain=dom,
         attrs=attrs,
         sparse=is_sparse,
-        allows_duplicates=is_sparse,
+        allows_duplicates=False,
```

Comment:
I would much prefer this was:
> allows_duplicates=create_options.get("allows_duplicates", False),

That would allow user control over this param, much like the other platform-dependent configuration.  I realize this _might_ allow the user to do sketchy things from a SOMA standpoint, but there are very legit use cases (mostly performance related, where datasets are known a priori to not contain dups)

Reply:
@bkmartinjr above I wrote

> # Notes for Reviewer:
> 
> Does not address https://github.com/single-cell-data/TileDB-SOMA/issues/876, which will be handled on a separate PR.

Reply:
I believe this answers your question -- ? Please let me know if it does not.

Reply:
I was making a statement - there is no question to answer :-)

I understand you are planning a separate PR.

Reply:
I thought we had to decided to keep allow_dups as non-parameterized and deal with overriding this via schema evolution? I'm not opposed to parameterizing it, though.

Reply:
I don't recall that decision - I thought it was only that the default (and implicit behavior) was 'no-dups'.  At least in our use case, there is a significant performance win from having a higher layer (app) take responsibility for the no-dups requirement, rather than doing it at the lower (TileDB) layer.  Allowing for that seems important.